### PR TITLE
feat(NDX-696): add jerk limit fields to MotionSettings and fix zero value blending bug

### PR DESCRIPTION
**Pull request by functionistic**
### Summary
- Add `joint_jerk_limits`, `tcp_jerk_limit`, and `tcp_orientation_jerk_limit` fields to `MotionSettings`, matching the new jerk fields in the API's `LimitsOverride`, `CartesianLimits`, and `JointLimits` models (experimental)
- Wire jerk limits through `as_limits_settings()`, `as_tcp_cartesian_limits()`, `as_joint_limits()`, and `has_limits_override()`
- Extend joint limits length validation to include `joint_jerk_limits`
- Fix `MotionSettings` checks so explicit `0` values are treated as set values instead of falsy/missing (subsumes #410)
- Add tests for zero-valued limits and blending edge cases

### Testing
```bash
PYTHONPATH=. uv run pytest -rs -v tests/nova/types/test_motion_settings.py
```

Closes #410

### CODE REVIEW

File: nova/types/motion_settings.py [hunk 1]
```diff
@@ -35,27 +35,40 @@ class MotionSettings(pydantic.BaseModel):
...
     blending_radius: float | None = pydantic.Field(default=None)
     joint_velocity_limits: tuple[float, ...] | None = pydantic.Field(default=None)
     joint_acceleration_limits: tuple[float, ...] | None = pydantic.Field(default=None)
+    joint_jerk_limits: tuple[float, ...] | None = pydantic.Field(default=None)
     tcp_velocity_limit: float | None = pydantic.Field(default=DEFAULT_TCP_VELOCITY_LIMIT)
     tcp_acceleration_limit: float | None = pydantic.Field(default=None)
+    tcp_jerk_limit: float | None = pydantic.Field(default=None)
     tcp_orientation_velocity_limit: float | None = pydantic.Field(default=None)
     tcp_orientation_acceleration_limit: float | None = pydantic.Field(default=None)
+    tcp_orientation_jerk_limit: float | None = pydantic.Field(default=None)
```
[the remark this answers is not in the document]
Comment (functionistic) [reply nova/types/motion_settings.py:71]: @copilot apply changes based on [this feedback](https://github.com/wandelbotsgmbh/wandelbots-nova/pull/416#discussion_r3155462539)

File: tests/nova/types/test_motion_settings.py [hunk 2]
```diff
@@ -69,3 +72,74 @@ def test_blending_settings():
...
+        MotionSettings(blending_radius=0.0, blending_auto=50)
+
+
+def test_zero_scalar_limits_are_treated_as_overrides():
+    """All scalar TCP limits should count as overrides even when explicitly set to zero."""
+    assert MotionSettings(tcp_velocity_limit=0.0).has_limits_override() is True
+    assert MotionSettings(tcp_acceleration_limit=0.0).has_limits_override() is True
+    assert MotionSettings(tcp_orientation_velocity_limit=0.0).has_limits_override() is True
+    assert MotionSettings(tcp_orientation_acceleration_limit=0.0).has_limits_override() is True
+
```
[the remark this answers is not in the document]
Comment (functionistic) [reply tests/nova/types/test_motion_settings.py:152]: @copilot apply changes based on [this feedback](https://github.com/wandelbotsgmbh/wandelbots-nova/pull/416#discussion_r3155462598)

File: nova/types/motion_settings.py [hunk 3]
```diff
@@ -68,10 +81,14 @@ def field_to_varname(cls, field):
         return f"__ms_{field}"
 
     def _get_blending_radius(self) -> float | None:
-        return self.blending_radius or self.position_zone_radius
+        if self.blending_radius is not None:
```
Comment (biering) [inline nova/types/motion_settings.py:84]: the position_zone_radius is now deprecated for a longer time, we could also think about removing it at one point

**Review by biering — APPROVED** [review]

File: nova/types/motion_settings.py (hunk 3 above)
Comment (functionistic) [reply nova/types/motion_settings.py:84]: NDX-709
